forge-game/src/main/java/forge/game/player/Player.java

```diff
@@ -1884,6 +1884,9 @@ public final Iterable<Player> getAttackedPlayersMyTurn() {
     public final List<Player> getAttackedPlayersMyLastTurn() {
         return attackedPlayersLastTurn;
     }
+    public final List<Player> getAttackedPlayersTheirCurrentTurn() {
```

Comment:
You just implemented the same function above?

Reply:
Removed this for now so this PR can be merged. Might take a shot at it later

Reply:
the property can just use `getAttackedPlayersMyTurn()`